tvtk/vtk_parser.py

```diff
@@ -295,7 +294,8 @@ def get_other_methods(self):
         """
         return self.other_meths
 
-    def get_method_signature(self, method):
+    @staticmethod
+    def get_method_signature(method):
```

Comment:
There are many occurrence of `parser = VTKMethodParser(); parser.get_method_signature(...)` in wrapper_gen and in tests (and possibly some other places I have not touched), could do with a separately cleanup PR in order to move this function out of the class?